### PR TITLE
fix: upgrade Fluentd to 1.14.4-sumo-1

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

**Comment by pmalek-sumo** [comment]
If you wait for https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2058 you'll have your ITs passing in this PR :)

**Comment by astencel-sumo** [comment]
> So it's a fix `Fixed` or fix `Changed`? 🤔

What are you referring to @pmalek-sumo?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- fix: upgrade Fluentd to 1.14.4-sumo-1 [#2057][#2057]
```
Comment (pmalek-sumo) [inline CHANGELOG.md:14]: I'm referring to this.

Commit says:

> fix: upgrade Fluentd to 1.14.4-sumo-1

but it's placed in `Changed` section

**Comment by swiatekm-sumo** [comment]
I think this should be a `chore`, that's what upgrades have been historically.

File: CHANGELOG.md (hunk 1 above)
Comment (astencel-sumo) [reply CHANGELOG.md:14]: I marked it as a "fix", because this upgrades is a security fix. But I can change it to `chore` as @swiatekm-sumo says.

**Pull request merged**